runhouse/rns/rns_client.py

```diff
@@ -664,20 +665,38 @@ def _get_logs(self):
         cleaned_log_lines = [ColoredFormatter.format_log(line) for line in log_lines]
         return " ".join(cleaned_log_lines)
 
-    async def send_cluster_logs_to_den(self, cluster_uri: str, api_server_url: str):
+    def _generate_logs_file_name(self):
```

Comment:
im not sure if `rns_client.py` is the best place for all these methods, feels like `http_utils.py` might be more appropriate?

Reply:
originally it was placed in `cluster_servlert.py`, which makes sense cause those functions (send_cluster_logs_to_den, send_status) are executed by a thread in the cluster servelet.

Maybe it is worth moving them back to `cluster_servlert.py`, wdyt? @jlewitt1

Reply:
Yea I think that works too